plugins/generic/pln/templates/ping.tpl

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Leading blank. Should also define a doctype and preferrably include a DTD or something that can be used to validate.